_episodes/04-recipe.md

```diff
@@ -368,7 +368,7 @@ Do you recognize the basic recipe structure that was introduced in episode 1?
 > >   which contains preprocessed data for each of the input datasets, plus a
 > >   combined `MultiModelMean`, a `metadata.yml` file and provenance files.
 > > - **map/script1**: creates `/run/map/script1` with general information and a
-> >   log of the diagnostic script run. It also creates `/plots/map/script1` and
+> >   log of the diagnostic script run. It also creates `/plots/map/script1/png` and
```

Comment:
If you want to add the "png" subfolder, you may do it also for the timeseries plots folder below.